.github/workflows/publish_docs.yml

```diff
@@ -16,10 +16,11 @@ jobs:
         with:
           node-version: 16
           cache: 'npm'
-          cache-dependency-path: docs/package-lock.json
+          cache-dependency-path: docs/pcakge-lock.json
```

Comment:
Looks like a typo here on the package lock file?

Reply:
Oh wait it's yarn, so we don't need this file. Could we just remove this entry then?

Reply:
Whoops. Good catch. I ended up reverting the `yarn` switch since we'd want to apply that to the `http` repo too.